data/ap214e3/AP214E3_2010.exp

```diff
@@ -4409,7 +4409,7 @@ WHERE
     item_identified_representation_usage.identified_item)) OR (
     'AUTOMOTIVE_DESIGN.' + 'DIMENSIONAL_SIZE' IN TYPEOF(SELF\
     item_identified_representation_usage.definition)) AND ((SELF\
-    item_identified_representation_usage.definition.name = 'heigth') OR (SELF\
+    item_identified_representation_usage.definition.name = 'height') OR (SELF\

```

Comment:
to quote @brlcad 
> On the surface, I'd think (hope) that this particular spelling fix can should be safe as it's in a string literal, not a protocol symbol. That said, don't know for certain so I'd suggest just adding one more line with the correct spelling instead of replacing the typo'd one in the meantime.

we should go with this backwards compatible change and update the upstream bug

Reply:
Some items for consideration:
1-What is the definition of backwards compatibility used here?
2-Which option is a transmitting system expected to pick?
3-This standard is not under active development and proposed spelling corrections will likely not be considered.
4-Apologies for tone of comment but changing the spelling of a string in a legacy AP is not likely to be widely appreciated.
5- Consequently, if I had a vote, I would vote to reject the merge request for the string change from 'heigth' to 'height'.

6- If the change is incorporated, the following should be added to the header of the schema:
"THIS SCHEMA HAS BEEN MODIFIED FROM THE SCHEMA DEFINED IN
ISO 10303-242 ed3  Core data for automotive mechanical design processes -- AUTOMOTIVE_DESIGN
AND SHOULD NOT BE INTERPRETED AS COMPLYING WITH THAT STANDARD."

7-The issue tracking was moved inside of the ISO internal development area because of concerns over the EU privacy laws, and the cost of supporting those laws for free sites.
8-step.bugzilla.com is a PDES inc. site that is a clone of the legacy wikistep site. It is an archival site as far as I know. PDES is a US org that develops ISO 10303 standards. You could send them an email.

Reply:
@TRThurman 
w.r.t. 2:  What are the transmitting systems using today?
w.r.t. 4:  Appreciated, maybe it's not our bug to fix in the first place.
w.r.t. 5:  It was proposed to add an additional conditional with the correct spelling, not remove the incorrect spelling.  Do you also have a problem with this?

w.r.t. 7 & 8: useful information, thanks